src/mono/browser/runtime/http.ts

```diff
@@ -72,12 +73,11 @@ export function http_wasm_create_controller (): HttpController {
     return controller;
 }
 
-function handle_abort_error (promise:Promise<any>) {
+function mute_unhandledrejection (promise:Promise<any>) {
     promise.catch((err) => {
         if (err && err !== "AbortError" && err.name !== "AbortError" ) {
-            Module.err("Unexpected error: " + err);
+            mono_log_debug("http muted" + err);
```

Comment:
Add a space between muted and err? And maybe a colon?

```suggestion
            mono_log_debug("http muted: " + err);
```